parfait-agent/src/main/java/io/pcp/parfait/ParfaitAgent.java

```diff
@@ -32,12 +32,23 @@ public static void setupProperties(String propertyAndValue, String separator) {
         }
     }
 
+    public static void loadContextProfile(GenericXmlApplicationContext context, String profile) {
+        context.getEnvironment().setActiveProfiles(profile);
+        context.load("classpath:agent.xml");
+        try {
+            context.load("file:/usr/share/parfait/*.xml");
+        } catch (Exception e) {
+            logger.trace("Cannot setup beans from /usr/share/parfait", e);
```

Comment:
Yes, IIRC it does error unfortunately, which was why I made it only trace level.